parquet/src/main/java/org/apache/iceberg/parquet/ParquetValueWriters.java

```diff
@@ -435,6 +514,11 @@ public void setColumnStore(ColumnWriteStore columnStore) {
     }
 
     protected abstract Iterator<Map.Entry<K, V>> pairs(M value);
+
+    @Override
+    public Stream<FieldMetrics> metrics() {
+      return Stream.of(keyWriter, valueWriter).flatMap(ParquetValueWriter::metrics);
```

Comment:
Why not `Stream.of(keyWriter.metrics(), valueWriter.metrics())`? That seems easier than creating a stream only to create another stream.